api/src/org/apache/cloudstack/api/command/admin/host/PrepareForMaintenanceCmd.java

```diff
@@ -108,4 +108,8 @@ public void execute() {
             throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, "Failed to prepare host for maintenance");
         }
     }
+
+    public void setHostId(final Long hostId) {
```

Comment:
I think the setters and getters are named per the variable, since the variable is hostId IDEs generate getHostId, setHostId named methods.